sphinx-docs/index.rst

```diff
@@ -0,0 +1,24 @@
+.. BELT (BERT For Longer Texts) documentation master file, created by
+   sphinx-quickstart on Fri May  5 13:19:47 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to BELT (BERT For Longer Texts)'s documentation!
+========================================================
+
+.. include:: bert_for_longer_texts.md
```

Comment:
After moving this file from docs to sphinx-docs, we must also update the link in the main README. Or maybe replace the part `More technical details are described in a [document](https://github.com/mim-solutions/bert_for_longer_texts/blob/main/docs/bert_for_longer_texts.md).` with the link to the documentation. What do you think?

Also, as Marek has suggested, we should copy the part about the package installation from the main README to the mainpage of the docs.

Reply:
1. link updated 
2. installation instructions included in the documentation